pages/add_remove_elem.py

```diff
@@ -0,0 +1,26 @@
+from playwright.sync_api import Page
+
+class AddRemoveElementsPage:
+    def __init__(self, page: Page):
+        self.page = page
+        self.url = "https://the-internet.herokuapp.com/add_remove_elements/"
+        self.selectors = _Selectors()
+
+    def visit(self):
+        self.page.goto(self.url)
```

Comment:
This should return, similar to the other functions in this file that are using playwright commands

Reply:
Done